source/disputes/index.html.md.erb

```diff
@@ -0,0 +1,95 @@
+| `lost`            | You have lost this dispute. The bank has decided the dispute in favour of the paying user.<br><br>The payment amount and your payment service provider’s dispute fee will be taken from the next payment to your bank account. You can see how much will be taken from your payout in the `net_amount` response attribute. |
+| `under_review`   | The bank has received evidence and is reviewing the disputed payment.|
+
+## Reasons for disputes
+
+When a paying user disputes a payment, they need to give their bank a reason for disputing the payment. You can see the reason the user gave in the `reason` response attribute.
+
+| `reason`                  | Meaning                                                                                                                                                                           |
+| --------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `credit_not_processed`  | The paying user claims they returned the product or cancelled the transaction, but you have not yet provided a refund or credit.                                                  |
+| `duplicate`               | The paying user claims you charged them more than once for the same product or service.                                                                                           |
+| `fraudulent`              | The paying user claims they did not authorise this payment.<br><br>This can happen if the card was lost or stolen, or the user does not recognise the payment.                    |
+| `general`                 | The bank has not categorised this dispute.<br><br>Contact the paying user to find out why they disputed this payment.                                                             |
+| `product_not_received`  | The paying user claims they did not receive the products or services they purchased.                                                                                              |
+| `product_unacceptable`   | The paying user claims the service they paid for was poor or not described accurately before they paid for it.                                                                    |
+| `unrecognised`            | The paying user claims they do not recognise the payment on their bank statement.                                                                                                 |
+| <nobr>`subscription_cancelled` | The paying user claims you continued to charge them after they cancelled their subscription to your service.                                                                      |
+| `other`                   | Your payment service provider has given an unexpected dispute reason.<br><br>[Contact us and we’ll investigate](/support_contact_and_more_information/#report-an-incident). |
```

Comment:
This links to `#report-an-incident`, but should it just link to the root of the support page?

Reply:
It should, yeah. No idea how that happened! Fixing in the next commit.